packages/config/src/layer2s/reddioex.ts

```diff
@@ -0,0 +1,159 @@
+import { EthereumAddress, ProjectId, UnixTime } from '@l2beat/shared-pure'
+
+import { ProjectDiscovery } from '../discovery/ProjectDiscovery'
+import {
+  getCommittee,
+  getProxyGovernance,
+  getSHARPVerifierContracts,
+  getSHARPVerifierGovernors,
+} from '../discovery/starkware'
+import { delayDescriptionFromString } from '../utils/delayDescription'
+import { formatSeconds } from '../utils/formatSeconds'
+import {
+  CONTRACTS,
+  DATA_AVAILABILITY,
+  EXITS,
+  FORCE_TRANSACTIONS,
+  makeBridgeCompatible,
+  NEW_CRYPTOGRAPHY,
+  NUGGETS,
+  OPERATOR,
+  RISK_VIEW,
+  STATE_CORRECTNESS,
+} from './common'
+import { Layer2 } from './types'
+
+const discovery = new ProjectDiscovery('reddioex')
```

Comment:
If you're using discovery you need to add your project to the `discovery` directory in the `backend` subpackage. The input file is `config.jsonc` and once you run `yarn discover ethereum reddioex` a `discovered.json` will be generated. Please include those two files in this PR!

Reply:
@mateuszradomski sure, just made the changes and committed already, thanks for the review and help.